Tests/SonarScanner.MSBuild.Shim.Test/PropertiesFileGeneratorTests.cs

```diff
@@ -404,9 +424,10 @@ public void GenerateFile_FilesOutOfProjectRootDir_TheyAreNotAnalyzedAndCorrectWa
 
         var projectDir = TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext, "project");
         var projectPath = Path.Combine(projectDir, "project.proj");
-        var projectInfo = TestUtils.CreateProjectInfoInSubDir(testDir, "project", null, Guid.NewGuid(), ProjectType.Product, false, "UTF-8", projectPath);
+        var projectInfo = TestUtils.CreateProjectInfoInSubDir(testDir, "project", null, Guid.NewGuid(), ProjectType.Product, false, projectPath, "UTF-8");
+        TestUtils.CreateEmptyFile(projectDir, "project.proj");
```

Comment:
Most of the tests needed this after the check.
I could clean this up more, but I think for now I prefer to not lose too much time and focus on the indexing issue.